doc/appendix-de-kassensichv/reference-tables/service-status-ftstate.md

```diff
@@ -1,14 +1,12 @@
-### Service Status: ftState
+# Service Status: ftState
 
-The table below describes supported statuses for the ftState field in accordance with the German law. These codes can be added through the logic operator "OR".
+The country-specific code is made of the country's code value following the ISO-3166-1-ALPHA-2 standard, converted from ASCII into hex. For Germany (DE) this is `0x4445`, which results in `0x4445000000000000` as the value for the "ready" status.
 
-The country-specific code is made of the country’s code value following the ISO-3166-1-ALPHA-2 standard, converted from ASCII into hex. For Germany (DE) this is `0x4445`, which results in `0x4445000000000000` as the value for the "ready" status.
-
-The table below describes supported statuses for the ftState field. Those codes can be combined by using the logic operator `OR`<span id="t-service-status-ftstate-22">.</span>
+The table below describes it supported statuses for the ftState field following German law. These codes can be added through the logic operator `OR`<span id="t-service-status-ftstate-22">.</span>
 
 | **Value**            | **Description**                                                                                     | **Middleware Version** |
 |----------------------|-----------------------------------------------------------------------------------------------------|---------------------|
-| `0x4445000000000002` | TSE device communication broken<br />The security mechanism was not able to communicate with the TSE device for at least one cycle. If this is the case no more communication trials are done to avoid long waiting times for each ReceiptRequest/ReceiptResponse sequence. To leave this state throw a Zero-Receipt, which forces a communication retry towards the TSE device.<br /> Receipts created in a state where no communication is possible with the TSE device are protected by the security mechanism of fiskaltrust. | 1.0                 |
+| `0x4445000000000002` | The security mechanism was not able to communicate with the TSE device for at least one cycle. If this is the case, no more communication trials are done to avoid long waiting times for each Receipt request/Receipt response sequence. To leave this state throw a Zero-Receipt, which forces a communication retry towards the TSE device.<br /> Receipts created in a state where no communication is possible with the TSE device are protected by the security mechanism of fiskaltrust. \| 1.0 | 1.0                 |
```

Comment:
i do not understand the sentence _To leave this state throw a Zero-Receipt_ . Does it mean that you are saying to the reader that he has to throw the receipt? If so, maybe you can rephrase it as "to leave this state, a zero-receipt must be thrown, which forces.." (although i don't know if the word "thrown" is what is needed here, could it be "produced" or "Created"?)